source/less/fonts.less

```
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: "Oswald";
+  font-weight: 500;
+  src: url("../fonts/oswaldmedium.woff2") format("woff2");
+  src: url("../fonts/oswaldmedium.woff") format("woff");
```

```suggestion
  src: url("../fonts/oswaldmedium.woff2") format("woff2"), url("../fonts/oswaldmedium.woff") format("woff");
```

В одном свойстве `src`, через запятую. Иначе, правило, которое идёт ниже, переопределит верхнее правило.